draft-ietf-cdni-uri-signing.xml

```diff
@@ -181,7 +181,9 @@
             <t>Redirection URI: URI created by the uCDN to redirect UA
             towards the Downstream CDN (dCDN). The Redirection URI can be signed by
             the uCDN and verified by the dCDN. In a cascaded
-            CDNI scenario, there can be more than one Redirection URI.</t>
+              CDNI scenario, there can be more than one Redirection URI.</t>
+            
+            <t>Signed Token Chain: A chain of signed JWTs that are used for subsequent access to a set of related resources in a CDN, such as a set of HTTP Adaptive Streaming files. Every time a signed JWT is used to access a particular resource, a new signed JWT is sent along with the resource that can be used to request the next resource in the set. When generating a new signed JWT in a Signed Token Chain, parameters are carried over from one signed JWT to the next.</t>
```

Comment:
Yes,  I didn't notice indeed because of the auto wrap. I'll format them properly.